source/installation.txt

```diff
@@ -0,0 +1,81 @@
+.. _scala-install:
+
+============
+Installation
+============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: dependency, reactive streams, add package
```

Comment:
Suggestion:
I think "sbt" and "maven" are two potentially helpful keywords to include.